### DISCUSSION TITLE
Technical Lead Team meeting September 2023

### DISCUSSION THREAD

**Discussion by ehogan** [ESMValGroup/Community#121] [Meetings]
Hi @ESMValGroup/technical-lead-development-team,

Here is a discussion for the next Tech Lead Team meeting, scheduled for Tuesday, September 5.

See [HackMD](https://hackmd.io/VceghTxTT5-qgYMkV4pAyA?both) for more details.

**Comment by valeriupredoi** [comment]
Many thanks for adding this @ehogan - I have now updated the agenda, and since we didn't have a September 5th call, we agreed to do it today :+1:
